### PR TITLE
setup db connection and package.json

### PR DESCRIPTION
**Pull request by yenting-biao**
I don't know whether my code follows the convention or not QQ

### CODE REVIEW

File: src/backend/index.js [hunk 1]
```diff
@@ -21,4 +23,39 @@ app.use("/api/book", bookRouter);
...
+await client.connect()
+  .then(() => {
+    // Successfully connected to PostgreSQL server
+    app.listen(port, () =>
+      console.log(`Server running on port http://localhost:${port}`),
+    );
+    console.log("Connected to PostgreSQL server");
+
+
+    app.get('/', async (req, res) => {
```
Comment (togimoto) [inline src/backend/index.js:45]: Add a comment to remind us to delete this

Comment (yenting-biao) [reply src/backend/index.js:45]: Wait I found that I somehow pushed a version without deleting this; I'll fix it later.

**Review by quevenliu — APPROVED** [review]
No problem.

**Comment by yenting-biao** [comment]
@quevenliu I fix all things. Please check it thank you!

**Pull request merged**